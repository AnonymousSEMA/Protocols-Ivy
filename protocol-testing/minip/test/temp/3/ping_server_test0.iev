> client_send_event({protocol:udp,addr:184549377,port:4990,interface:ivy},{protocol:udp,addr:184549378,port:4443,interface:ivy},1)
> frame.ping.handle({data:[112,105,110,103]})
< show_current_time(0x5f)
> frame.timestamp.handle({time:0x2})
> ping_packet_event({protocol:udp,addr:0xb000001,port:0x137e,interface:ivy},{protocol:udp,addr:0xb000002,port:0x115b,interface:ivy},{payload:[{frame.ping:{data:[0x70,0x69,0x6e,0x67]}},{frame.timestamp:{time:0x82}}]})
< recv_pong(client,{protocol:udp,addr:0xb000002,port:0x115b,interface:ivy},{protocol:udp,addr:0xb000001,port:0x137e,interface:ivy},{payload:[{frame.pong:{data:[0x70,0x6f,0x6e,0x67]}},{frame.timestamp:{time:0x6acfd1ff}}]})
< show_current_time(0x1206)
assumption_failed("/usr/local/lib/python2.7/dist-packages/ms_ivy-1.8.24-py2.7.egg/ivy/include/1.7/ping_frame.ivy: line 96")
assumption_failed("require enough_wait;")
