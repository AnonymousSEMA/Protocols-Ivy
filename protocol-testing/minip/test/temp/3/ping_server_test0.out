Received message {1: ping, 3: 0}
Sending Pong...
Pong sent
